profile/README.md

```diff
@@ -21,17 +21,22 @@ We 💙 open source development. If you're here, chances are you do too! Contrib
 -----------------------------------------------------------------
 
 <p align="center">
-  <img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExYm9odzU4NTlib2x5cHA0bWEwcXJ6M3Fxano5OXd4eDVyZ2dxem5tYyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/AC4ncZp0zV7seSfU9y/giphy.gif" alt="Center Aligned Image" />
-</p>
-
+  <img width = 450px src="https://github.com/varun10sudhir/.github/blob/main/profile/github.png" alt="Source Sprint" />
```

Comment:
Didn't notice this. Please make this change. Also, change the filename to `source-sprint.png` instead of `github.png`
```suggestion
  <img width = 450px src="https://github.com/IEEE-VIT/.github/blob/main/profile/source-sprint.png" alt="Source Sprint" />
```